agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -1501,7 +1501,10 @@ bool monitor_thread::handle_ap_metrics_query(Socket &sd, ieee1905_1::CmduMessage
         const auto &bssid = std::get<1>(bssid_tuple);
         LOG(DEBUG) << "Received AP_METRICS_QUERY_MESSAGE, mid=" << std::hex << int(mid)
                    << "  bssid " << bssid;
-        if (!mon_stats.add_ap_metrics(cmdu_tx, bssid)) {
+        if (!mon_stats.add_ap_metrics(cmdu_tx, bssid, mon_db.get_sta_count(),
```

Comment:
mon_db.get_sta_count is incorrect, it should only be the stations associated with the VAP.

Reply:
Fixed